test/framework/fixtures/restClientFixture.go

```diff
@@ -228,15 +224,15 @@ func (f *RestClientFixture) getNodeWalletsSortedByBalance(client libgoal.Client)
 // WaitForTxnConfirmation waits until either the passed txid is confirmed
 // or until the passed roundTimeout passes
 // or until waiting for a round to pass times out
-func (f *RestClientFixture) WaitForTxnConfirmation(roundTimeout uint64, accountAddress, txid string) bool {
-	_, err := f.WaitForConfirmedTxn(roundTimeout, accountAddress, txid)
+func (f *RestClientFixture) WaitForTxnConfirmation(roundTimeout uint64, txid string) bool {
+	_, err := f.WaitForConfirmedTxn(roundTimeout, txid)
 	return err == nil
 }
 
 // WaitForConfirmedTxn waits until either the passed txid is confirmed
 // or until the passed roundTimeout passes
 // or until waiting for a round to pass times out
-func (f *RestClientFixture) WaitForConfirmedTxn(roundTimeout uint64, accountAddress, txid string) (txn v2.PreEncodedTxInfo, err error) {
+func (f *RestClientFixture) WaitForConfirmedTxn(roundTimeout uint64, txid string) (txn v2.PreEncodedTxInfo, err error) {
```

Comment:
You're worried that we are fetching the wrong transaction?  Or that there's a hash collision and some other txn has the same id but different sender?  This seems like very weird thing to imagine happening.

Would you advocate that we pass a bunch of other stuff we know about the transaction and check them for equality too?  Feels like a historic artifact we would never be testing if it didn't happen coincidentally because of an old API.